run-integration-tests.sh

```diff
@@ -1,6 +1,6 @@
-./run src/examples/zkapps/hello_world/run.ts
-./run src/examples/zkapps/voting/run.ts
-./run src/examples/simple_zkapp.ts
-./run src/examples/zkapps/reducer/reducer_composite.ts
-./run src/examples/zkapps/composability.ts
-./run src/examples/zkapps/token_with_proofs.ts
+./run src/examples/zkapps/hello_world/run.ts || exit 1
```

Comment:
change to make sure that the intg tests really made CI fail if one of them failed

Reply:
This is great!